.github/CODEOWNERS

```diff
@@ -1115,11 +1115,17 @@
 #PRLabel: %CodeGen %Client
 /eng/packages/http-client-csharp/                                  @JoshLove-msft @m-nash @jorgerangel-msft @live1206 @ArcturusZhang
 
+#PRLabel: %CodeGen %Mgmt
+/eng/packages/http-client-csharp-mgmt/                                  @JoshLove-msft @m-nash @jorgerangel-msft @live1206 @ArcturusZhang
+
 # ServiceLabel: %CodeGen
 # AzureSdkOwners:                                                  @JoshLove-msft
 
 #PRLabel: %CodeGen %Client
 /eng/packages/http-client-csharp/generator/Azure.Generator/        @JoshLove-msft @m-nash @jorgerangel-msft @jsquire @live1206 @ArcturusZhang
 
+#PRLabel: %CodeGen %Mgmt
+/eng/packages/http-client-csharp-mgmt/generator/Azure.Generator.Mgmt/        @JoshLove-msft @m-nash @jorgerangel-msft @jsquire @live1206 @ArcturusZhang
```

Comment:
There is no value to this entry, as the block on L1118 will cover it.   Please remove.

Reply:
removed